Ardupilog.m

```diff
@@ -132,6 +133,9 @@ function delete(obj)
                     continue;
                 end
 
+                msgLen = obj.fmt_cell{i,3};
+                data = obj.isolateMsgData(msgId,msgLen,allHeaderCandidates);
+
```

Comment:
Ah, I see that you need to fill the valid_msgheader_cell. Okay.
Would it be possible to avoid that, to avoid unnecessary cycles?